roles/lib_openshift_api/library/oc_secrets.py

```diff
@@ -0,0 +1,360 @@
+#!/usr/bin/env python
+'''
+module for openshift cloud secrets
+'''
+#   Examples:
+#
+#  # to initiate and use /etc/origin/master/admin.kubeconfig file for auth
```

Comment:
@detiber,  You can specify a namespace for the secrets so I believe this is accurate.